evergreen.yml

```diff
@@ -65,17 +65,23 @@ functions:
         shell: bash
         working_dir: mongo-powerbi-connector
         script: |
-          export MONGO_POWERBI_CONNECTOR_VERSION=$(git describe --always | sed 's/^v//g')
           export UPDATE_PLUGIN_VERSION=true
           git describe --exact-match HEAD || UPDATE_PLUGIN_VERSION=false
 
+          if [[ "${triggered_by_git_tag}" != "" ]]; then
+            export release_version=$(echo ${triggered_by_git_tag} | sed s/v//)
+          else
+            export release_version=test_release
```

Comment:
Since the connector doesn't build with a non semver version, how about we set this to something like 0.0.0-`output from git describe`.  It isn't used in `build connector` at the moment but I'm thinking if there's a mistake or confusion it might be used.

Reply:
Sounds good. I set it to be `0.0.0-<branch>`, but maybe we want the commit?

Reply:
Yeah, I was thinking of something with the commit as well.  `git describe --always` should work.